docs/docs/llms/llm-setup.mdx

```diff
@@ -70,14 +79,19 @@ to be configured:
   OpenAI Service.
 - `openai.api_base`: This should be the URL for your Azure OpenAI instance. An
   example might look like this: "https://docs-test-001.openai.azure.com/".
-- `openai.api_version`: This should be the version of Azure OpenAI you are
-  using, such as "2023-05-15".
+
 
 To configure these parameters, follow these steps:
 
-1. [Instructions to set `openai.api_type`]
-2. [Instructions to set `openai.api_base`]
-3. [Instructions to set `openai.api_version`]
+1. To configure the `openai.api_type` as an environment variable:
```

Comment:
can we describe what "api_type" means using words? also, what are other valid options we support?

Reply:
oh ok, I didn't see that, then ignore me